### PR TITLE
visp 3.2.0

### PR DESCRIPTION
**Pull request by fspindle**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/visp.rb [hunk 1]
```diff
@@ -1,30 +1,22 @@
 class Visp < Formula
   desc "Visual Servoing Platform library"
   homepage "https://visp.inria.fr/"
-  url "https://gforge.inria.fr/frs/download.php/latestfile/475/visp-3.1.0.tar.gz"
-  sha256 "2a1df8195b06f9a057bd4c7d987697be2fdcc9d169e8d550fcf68e5d7f129d96"
-  revision 2
-
-  bottle do
```
Comment (Moisan) [inline Formula/visp.rb]: Please do not change the bottle block, it will be updated automatically when we merge the pull-request.

**Review by Moisan — CHANGES_REQUESTED** [review]

File: Formula/visp.rb [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
+  sha256 "072237ed5c6fcbc6a87300fa036014ec574fd081724907e41ae2d6fb5a222fbc"
 
+  # Introduce pkg-config and glew dependencies to avoid cmake issues when checking for pcl
   depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
   depends_on "eigen"
+  depends_on "glew"
   depends_on "gsl"
   depends_on "jpeg"
+  depends_on "lapack"
```
Comment (fxcoudert) [inline Formula/visp.rb]: lapack is provided by macOS, we should build against that

File: Formula/visp.rb [hunk 3]
```diff
@@ -1,30 +1,22 @@
...
-
-  bottle do
-    sha256 "ed11d41df8e68b67795cdde1cc9dc2761bd66b6cb406e0cbc955db90ff066a9e" => :mojave
-    sha256 "a2b5b96e25a69762a1a6d07ab66bf9790adc9fc436a5a57c29d82f9ca13c426f" => :high_sierra
-    sha256 "b157bf06ff32d11195abafc2e82b7277e08b76bddeff95690f7a7277735ca118" => :sierra
-  end
+  url "https://gforge.inria.fr/frs/download.php/latestfile/475/visp-3.2.0.tar.gz"
+  sha256 "072237ed5c6fcbc6a87300fa036014ec574fd081724907e41ae2d6fb5a222fbc"
 
+  # Introduce pkg-config and glew dependencies to avoid cmake issues when checking for pcl
```
Comment (fxcoudert) [inline Formula/visp.rb]: We should not add dependencies unless they are needed for function, and we understand why.

File: Formula/visp.rb [hunk 4]
```diff
@@ -48,14 +40,17 @@ def install
...
+                         "-DLAPACK_INCLUDE_DIR=#{Formula["lapack"].opt_include}",
+                         "-DLAPACK_LIBRARY=#{Formula["lapack"].opt_lib}/liblapack.dylib",
                          "-DUSE_LIBUSB_1=OFF",
                          "-DUSE_OPENCV=ON",
                          "-DOpenCV_DIR=#{Formula["opencv"].opt_share}/OpenCV",
                          "-DUSE_PCL=OFF",
                          "-DUSE_PNG=ON",
                          "-DPNG_PNG_INCLUDE_DIR=#{Formula["libpng"].opt_include}",
                          "-DPNG_LIBRARY_RELEASE=#{Formula["libpng"].opt_lib}/libpng.dylib",
+                         "-DUSE_PCL=OFF",
```
Comment (fxcoudert) [inline Formula/visp.rb]: This is already above, so there is clearly something weird here

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/visp.rb (hunk 4 above)
Comment (fspindle) [reply Formula/visp.rb]: Should be ok now

**Pull request closed without merging**